assets/wizards/connections/views/main/recaptcha.js

```diff
@@ -95,18 +99,47 @@ const Recaptcha = () => {
 				{ settings.use_captcha && (
 					<>
 						{ error && <Notice isError noticeText={ error } /> }
-						{ settings.use_captcha && ( ! settings.site_key || ! settings.site_secret ) && (
+						{ ! hasRequiredSettings && (
 							<Notice
 								noticeText={ __(
 									'You must enter a valid site key and secret to use reCAPTCHA.',
 									'newspack-plugin'
 								) }
 							/>
 						) }
+						<Grid noMargin rowGap={ 16 }>
+							<BaseControl
+								id="recaptcha-version"
+								label={ __( 'reCAPTCHA Version', 'newspack-plugin' ) }
+								help={
+									<ExternalLink href="https://developers.google.com/recaptcha/docs/versions">
+										{ __( 'Learn more about reCAPTCHA versions', 'newspack-plugin' ) }
+									</ExternalLink>
+								}
+							>
+								<SelectControl
+									label={ __( 'reCAPTCHA Version', 'newspack-plugin' ) }
+									hideLabelFromVision
+									value={ settingsToUpdate?.version || 'v3' }
+									onChange={ value => {
+										console.log( value );
```

Comment:
Should be removed (we need to update eslint config!)

Reply:
Whoops, thanks for catching! Removed in 8dd15857